screens/Scan/SuccessfullySharedModal.tsx

```diff
@@ -41,13 +42,38 @@ export const SharingSuccessModal: React.FC<
             {sendVcScreenController.receiverInfo.name}
           </Text>
         </Column>
-        <Column margin="0 0  0">
-          <Button
-            type="gradient"
-            title={t('ScanScreen:status.accepted.gotohome')}
-            onPress={scanLayoutController.DISMISS}
-          />
-        </Column>
+        <Row
+          align="space-evenly"
+          style={{marginBottom: Dimensions.get('screen').height * 0.06}}>
+          <Pressable>
+            <Icon
+              name="home"
+              color={Theme.Colors.Icon}
+              size={33}
+              containerStyle={
+                Theme.SelectVcOverlayStyles.sharedSuccessfullyIconStyle
+              }
+              onPress={scanLayoutController.DISMISS}
```

Comment:
Can we change this event to `scanLayoutController.DISMISS` -> `scanLayoutController.GO_TO_HOME` as we are navigating to home here

Reply:
Yes, we can. But better to not change right now. Since most of the places in the scanMachine we are using same **DISMISS** event.
For this we may need to make changes in many places which will affect the flow of sharing VC.